### PR TITLE
Update contributing documentation

### PR DESCRIPTION
**Pull request by datho7561**
Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,11 +41,92 @@ There are only a few guidelines that we need contributors to follow.
...
+
+The Chrome debug tools can also be used to help with writing the layout of webviews.
+In order to inspect the layout, switch to the "Elements" view.
+Then, you will need to walk through the tree in order to locate the webview HTML.
+It should be nested in two `<iframe>` elements (in order to sandbox it from the rest of VS Code).
+It will look something like this:
+
+![VS Code with the "Create Component" webview open. On the right, the Chrome debug tools are open, showing a tree that represents the HTML for the webview. In VS Code, the "Create Component" webview is highlighted in blue to indicate it's being hovered over in the Chrome debug tools HTML tree view](images/chrome-devtools-webview.png)
+
+Then, you can modify the CSS and attributes of elements in order to help experiment with or debug the webview layout.
```
Comment (JessicaJHee) [inline CONTRIBUTING.md:76]: help experiment with ...

**Review by JessicaJHee — COMMENTED** [review]
This is a super nice addition, thank you for taking the time to do this! I learned a lot reading this :) LGTM other than one small thing

File: CONTRIBUTING.md (hunk 1 above)
Comment (datho7561) [reply CONTRIBUTING.md]: TBH I'm not really sure what you mean by this, but I tried to clean up the wording, so let me know what you think

Comment (JessicaJHee) [reply CONTRIBUTING.md]: Ohh sorry I thought there was supposed to be something after 'with', but looks good to me now!

Comment (datho7561) [reply CONTRIBUTING.md]: Okay!

**Comment by datho7561** [comment]
Thanks for the review, Jessica!

**Pull request merged**